src/morphology_workflows/curation.py

```diff
@@ -52,7 +52,7 @@ def collect(row, data_dir, morph_path_col="morph_path"):
         )
     is_morph, ext = is_morphology(row[morph_path_col])
     if is_morph:
-        full_new_morph_path = (data_dir / row.name).with_suffix(ext)
+        full_new_morph_path = str(data_dir / row.name) + ext
```

Comment:
I don't understand this change, what's the difference?

Reply:
if a name of morpho has a . it cuts it

Reply:
Are you sure about this?
This:
```python
Path("/tmp/toto.tata.titi.test").with_suffix(".truc")
```
returns `PosixPath('/tmp/toto.tata.titi.truc')`, which looks correct to me, isn't it?

Reply:
yes, sneaky :P